docs/external_system.md

```diff
@@ -99,33 +99,33 @@ use externally-accessible API endpoints exposed by CSM.
       csm-api-client@git+https://github.com/Cray-HPE/python-csm-api-client@release/1.1
       ```
 
-   1. (Optional) Confirm that `requirements.lock.txt` is modified as expected.
+   1. (Optional) (`venv`) Confirm that `requirements.lock.txt` is modified as expected.
 
       **Note:** For versions newer than 3.19, you will see both `cray-product-catalog`
       and `csm-api-client`. For version 3.19 and older, you will only see
       `cray-product-catalog`.
 
       ```screen
-      (venv) $ grep -E 'cray-product-catalog|csm-api-client' sat/requirements.lock.txt
+      $ grep -E 'cray-product-catalog|csm-api-client' sat/requirements.lock.txt
       ./cray-product-catalog
       csm-api-client@git+https://github.com/Cray-HPE/python-csm-api-client@release/1.1
       ```
 
-1. Install the modified SAT dependencies.
+1. (`venv`) Install the modified SAT dependencies.
 
    ```screen
-   (venv) $ pip install -r sat/requirements.lock.txt
+   $ pip install -r sat/requirements.lock.txt
    ...
```

Comment:
Remove this line that is supposed to be showing truncated output from the command. We want just copy-pastable commands in our code blocks now.

Reply:
Updated in next commit.

Reply:
Updated in next commit.